src/test/java/com/ncc/aif/ExamplesAndValidationTest.java

```diff
@@ -1508,12 +1516,162 @@ void invalidMultipleHandles() {
             @Test
             // One handle on entity cluster in hypothesis
             void valid() {
-                makeHypothesis(model, getUri("hypothesis-1"), Collections.singleton(entity), system);
-                testValid("NISTHypothesis.invalid: Each entity cluster in the hypothesis graph must have " +
+                markImportance(makeHypothesis(model, getUri("hypothesis-1"),
+                        Collections.singleton(entity), system), 100);
+                testValid("NISTHypothesis.valid: Each entity cluster in the hypothesis graph must have " +
                         "exactly one handle");
             }
         }
 
+        // Each hypothesis graph must have exactly one hypothesis importance value
+        @Nested
+        class HypothesisImportanceValue {
+            @Test
+            void invalid() {
+                //invalid hypothesis, no importance value
+                makeHypothesis(model, getUri("hypothesis-1"), Collections.singleton(entity), system);
+                testInvalid("NISTHypothesis.invalid: Each hypothesis graph must have exactly one" +
+                        " hypothesis importance value");
+            }
+            @Test
+            void valid() {
+                markImportance(makeHypothesis(model, getUri("hypothesis-1"),
+                        Collections.singleton(entity), system), 100);
+                testValid("NISTHypothesis.valid: Each hypothesis graph must have exactly one" +
+                        " hypothesis importance value");
+            }
+        }
+
+        // Each event or relation (cluster) in the hypothesis must have exactly one importance value
+        @Nested
+        class HypothesisEntityRelationClusterImportanceValue {
+
+            private final String documentEventUri = getUri("ldc:event-1");
+            private final String relationUri = getUri("ldc:relation-1");
+            Resource event;
+            Resource relation;
+
+            @BeforeEach
+            void setup() {
```

Comment:
I like the way this class builds up from invalid to valid. Maybe that's a paradigm that would work well in solving Craig's problem